bucket/audmes.json

```
@@ -0,0 +1,22 @@
+{
+    "version": "20211104",
+    "description": "AUDio MEasurement System - PC based Oscilloscope and Spectrum analyzer using sound card",
+    "homepage": "https://sourceforge.net/projects/audmes/",
+    "license": "GPLv2",
+    "url": "https://phoenixnap.dl.sourceforge.net/project/audmes/audmes-win32%20binary/2021-11-04/AudMeS-20211104-win32.zip",
+    "hash": "sha256:ca6da925ae4fdd63a2cc0c92901923c2434312a4617170a50db0858960968a73",
+    "bin": "AudMeS-0.1.1-win32/AudMeS.exe",
+    "shortcuts": [
+        [
+            "AudMeS-0.1.1-win32/AudMeS.exe",
+            "AudMeS"
+        ]
+    ],
+    "checkver": {
+        "url": "https://sourceforge.net/projects/audmes/rss?path=/",
+        "regex": "(?<pathVersion>[\\d-]+)/AudMeS-(?<version>[\\d.]+)-win32\\.zip"
+    },
+    "autoupdate": {
+        "url": "https://sourceforge.net/projects/audmes/files/audmes-win32%20binary/$pathVersion/AudMeS-$version-win32.zip/download"
```

```suggestion
        "url": "https://downloads.sourceforge.net/projects/audmes/files/audmes-win32%20binary/$matchPathversion/AudMeS-$version-win32.zip"
```
* match groups are **$matchXXX** in *autoupdate* section, except for `?<version>` see [our Wiki](https://github.com/ScoopInstaller/Scoop/wiki/App-Manifest-Autoupdate#add-autoupdate-to-a-manifest) for details.

* Your url is actually re-directed to `downloads.sourceforge.net/....../AudMeS-20211104-win32.zip`. You can check the download tab in your browser, or use *WGet* (`scoop install wget`) to find it out.